### DISCUSSION TITLE
Datepicker: Allow setting Monday as the first day of the week

### DISCUSSION THREAD

**Discussion by timobehrens** [primefaces/community#2512] [PrimeVue] [answered]
It would be awesome if the Datepicker component displays the respective first day of the week correctly based on the users locale. Currently the week starts on Sunday in the component.

**Comment by melloware** [comment, marked as answer]
i believe its controlled by your locale set in PrimeVue.

See PrimeLocale: https://github.com/primefaces/primelocale

Value in the locale: `"firstDayOfWeek": 0,` or 1 in your case.

Here is a working demo: https://codesandbox.io/p/devbox/primevue-locale-loader-forked-mxkhh8?file=%2Fsrc%2FApp.vue

Try picking your language and see if it works?

**Reply by timobehrens** [reply]
Thanks! I have not seen any reference or indication about the first day of week anywhere in the doc or in that primelocale module.

**Reply by timobehrens** [reply]
I am wondering why primeLocale is not part of the core package. Wouldn't it make sense?

**Reply by melloware** [reply]
PrimeLocale is used in PrimeNg PrimeReact PrimeVue and PrimeFaces so it is a reusable NPM package. 

If you mean why isn't it a "forced" dependency of PrimeVue that is a question for PrimeTek.
